### PR TITLE
Add direct access to frame pict type in Frame Class

### PR DESCRIPTION
**Pull request by steeveen**
[https://github.com/bytedeco/javacv/issues/1729](url)

### CODE REVIEW

**Comment by saudet** [comment]
Instead of using `int` or even `enum`, I was thinking we could just use `char` with  as values `I`, `P`, `B`, and maybe others in the future. What do you think?

**Comment by steeveen** [comment]
Not a big problem.
`char` seems to be better for readability
But I saw the type contains SI、SP、BI
image
Using String to represent them might be not a appropriate way.

**Comment by saudet** [comment]
Looks like FFmpeg itself is just using lowercase letters in that case:
https://github.com/FFmpeg/FFmpeg/blob/master/libavutil/utils.c#L83-L95
So, let's just call `av_get_picture_type_char()` to get a `char` and put that in :)

**Comment by steeveen** [comment]
OK，you are right. I have converted it ~

File: src/main/java/org/bytedeco/javacv/Frame.java [hunk 1]
```diff
@@ -128,6 +132,7 @@ public Frame(int width, int height, int depth, int channels, int imageStride) {
         this.image = new Buffer[1];
         this.data = null;
         this.streamIndex = -1;
+        this.pictType = (char) av_get_picture_type_char(AV_PICTURE_TYPE_NONE);
```
Comment (saudet) [inline src/main/java/org/bytedeco/javacv/Frame.java:135]: Let's just set that to 0 here to prevent a dependency on FFmpeg.

Comment (steeveen) [reply src/main/java/org/bytedeco/javacv/Frame.java:135]: `av_get_picture_type_char` also depend on FFmpeg. 
How about init pictType with `'N'` or just `''`

Comment (saudet) [reply src/main/java/org/bytedeco/javacv/Frame.java:135]: `''` isn't a valid char literal, you probably mean `'\0'`, which is the same as just `0`.

Comment (steeveen) [reply src/main/java/org/bytedeco/javacv/Frame.java]: Got it! Done.

**Pull request merged**